### PR TITLE
#19/🐰: sorry console appear with sorry text in it whyle user resize the window

### PR DESCRIPTION
**Pull request by elkinny**
**What was done**
When user resize browser window and reach min width, Sorry console appear with sorry text in it

**Issue was fixed**
Bug #19 was fixed

### CODE REVIEW

File: src/js/console/console.js [hunk 1]
```diff
@@ -11,27 +11,25 @@ export default class Console {
...
-                            this.words.push(this.words.shift());
-                            this.x = 1;
-                            letterCount += this.x;
-                            this.waiting = false;
-                        }, 1000);
-                    } else if (letterCount === this.words[0].length + 1 && this.waiting === false) {
-                        this.x = -1;
+        window.setInterval(() => {
+            if (document.body.clientWidth <= (400 * 2)) { //$floorWidth
+                if (letterCount === 0 && this.waiting === false) {
```
Comment (Dmitry-White) [inline src/js/console/console.js]: `this.waiting === false` can be expressed as `!this.waiting`

File: src/js/console/console.js [hunk 2]
```diff
@@ -11,27 +11,25 @@ export default class Console {
...
+                        this.words.push(this.words.shift());
+                        this.x = 1;
                         letterCount += this.x;
-                    } else if (this.waiting === false) {
-                        this.target.innerHTML = this.words[0].substring(0, letterCount)
-                        letterCount += this.x;
-                    }
+                        this.waiting = false;
+                    }, 1000);
+                } else if (letterCount === this.words[0].length + 1 && this.waiting === false) {
```
Comment (Dmitry-White) [inline src/js/console/console.js]: Same as above

File: src/js/console/console.js [hunk 3]
```diff
@@ -11,27 +11,25 @@ export default class Console {
...
+                    this.x = -1;
+                    letterCount += this.x;
+                } else if (this.waiting === false) {
```
Comment (Dmitry-White) [inline src/js/console/console.js]: You guessed it, right?)

File: src/js/console/console.js (hunk 1 above)
Comment (elkinny) [reply src/js/console/console.js]: prefer a precise definition

File: src/js/console/console.js (hunk 2 above)
Comment (elkinny) [reply src/js/console/console.js]: prefer a precise definition

File: src/js/console/console.js (hunk 3 above)
Comment (elkinny) [reply src/js/console/console.js]: prefer a precise definition

**Pull request merged**